easybuild/easyconfigs/q/Qt/Qt-4.8.7_openssl_1.1.patch

```diff
@@ -1,50 +1,54 @@
 Description: Compile with openssl-1.1.0
- * Most changes are related to openssl structures are now opaque. 
+ * Most changes are related to openssl structures are now opaque.
  * The network/ssl threading setup has been disabled because the
    old openssl threading model has been removed and is apparently
    no longer needed.
  * A number of new functions had to be imported (see changes to
-   src/network/ssl/qsslsocket_openssl_symbols.cpp) 
+   src/network/ssl/qsslsocket_openssl_symbols.cpp)
 Author: Gert Wollny  <gw.fossdev@gmail.com>
 Author: Dmitry Eremin-Solenikov <dbaryshkov@gmail.com>
 Last-Update: 2017-11-07
 Bug-Debian: http://bugs.debian.org/828522
 
----
- src/network/ssl/qsslcertificate.cpp            |   25 +++--
- src/network/ssl/qsslkey.cpp                    |   11 ++
- src/network/ssl/qsslsocket_openssl.cpp         |   27 ++++-
- src/network/ssl/qsslsocket_openssl_p.h         |    4 
- src/network/ssl/qsslsocket_openssl_symbols.cpp |  118 +++++++++++++++++++++++++
- src/network/ssl/qsslsocket_openssl_symbols_p.h |   76 ++++++++++++++++
- 6 files changed, 249 insertions(+), 12 deletions(-)
+Adapted to build on OpenSSL 1.0.2k (on CentOS 7.4) by https://github.com/crubb (2018-08-25)
```

Comment:
@crubb Since you modified the original patch, we should highlight that a bit more, by:

* moving this line up to the top of the patch
* include a short description of what you had to change in the patch to avoid problems with OpenSSL 1.0.2k...